src/mctpd.c

```diff
@@ -3035,6 +3365,18 @@ static int mctpd_dbus_enumerate(sd_bus *bus, const char* path,
 		if (ctx->peers[i].published)
 			num_nodes++;
 
+	// .../mctp1/interfaces object
+	num_nodes++;
+
+	// .../mctp1/interface/<name>
+	for (i = 0; i < ctx->nl->linkmap_count; i++) {
```

Comment:
The linkmap definition should be private to netlink.c. Instead of this, use `mctp_nl_if_list` and iterate over the results of that.